bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -507,10 +503,6 @@
 		<label>Precipitation</label>
 		<description>Precipitation volume of the related timespan.</description>
 		<category>Rain</category>
-		<tags>
-			<tag>Measurement</tag>
-			<tag>Rain</tag>
```

Comment:
Which rule do these tags break? Is it because it implicitly refers to an (undefined) measurement period?

Reply:
In fact, it is forecasted precipitations. It is a channel type only member of group named oneCallMinutely having this label "One Call API Minutely Forecast". Maybe the channel label or description could be improved.

Reply:
Ok, I indeed didn't notice this as I didn't look at the channel group of it. Thanks for the explanation!

Reply:
And you have another channel type "rain" with tags Measurment + Rain.